src/test/java/plugins/WorkflowPluginTest.java

```diff
@@ -196,30 +201,49 @@
 
     @Category(DockerTest.class)
     @WithDocker
-    @WithPlugins({"workflow-aggregator@1.14", "docker-workflow@1.4", "git", "ssh-agent@1.10"})
+    @WithPlugins({"workflow-cps@2.0", "workflow-job@2.0", "workflow-durable-task-step@2.0", "docker-workflow@1.5", "git", "ssh-agent@1.10", "ssh-slaves@1.11"})
     @WithCredentials(credentialType=WithCredentials.SSH_USERNAME_PRIVATE_KEY, values={"git", "/org/jenkinsci/test/acceptance/docker/fixtures/GitContainer/unsafe"}, id="gitcreds")
     @Issue("JENKINS-27152")
     @Test public void sshGitInsideDocker() throws Exception {
-        // Pending https://github.com/jenkinsci/docker-workflow-plugin/pull/31 (and additional APIs in DockerFixture etc.)
-        // it is not possible to run this on a bind-mounted Docker slave, so we cannot verify that paths are local to the slave.
-        GitContainer container = gitServer.get();
-        String host = container.host();
-        int port = container.port();
+        GitContainer gitContainer = gitServer.get();
         GitRepo repo = new GitRepo();
         repo.commit("Initial commit");
-        repo.transferToDockerContainer(host, port);
+        repo.transferToDockerContainer(gitContainer.host(), gitContainer.port());
+        DumbSlave slave = jenkins.slaves.create(DumbSlave.class);
+        slave.setExecutors(1);
+        slave.remoteFS.set("/home/test"); // TODO perhaps should be a constant in SshdContainer
+        SshSlaveLauncher launcher = slave.setLauncher(SshSlaveLauncher.class);
+        Process proc = new ProcessBuilder("stat", "-c", "%g", "/var/run/docker.sock").start();
```

Comment:
Works for me on Linux; might cause the test to be skipped on some environments (e.g., boot2docker), but I cannot test these.